### DISCUSSION TITLE
Problem with translation in frontend of JCB plugin.

### DISCUSSION THREAD

**Discussion by saman222** [joomengine/Joomla-Component-Builder#782] [Q&A] [answered]
hi
1-when implement a plugin with Siteview and use the JText for render translation, then it import to "Language Translation" menu  of JCB and also import to backend .ini file in the public_html/administrator/language/*** rather than public_html/languages/*** like language translation of component. 

for example i create a plugin for onuserathunticate  and i want to enqueue message for front end user for example say their "your password is wrong" but now i can not use JText to implement that and it just appear in the backend interface of plugin.
i hope you can understand what i say.

2- also i have another question don't understand it is problem or not. in JCB when i use _ for label of the JText it can not import to "Language Translation" and must use without _ for example i must use JText("COM MY COMPONENT") instead of JText("COM_MY_COMPONENT") . is it normally?
thank you

**Comment by Llewellynvdm** [comment, marked as answer]
I will have to take a closer look, and will get back to you.
